src/App.js

```diff
@@ -1,11 +1,13 @@
-import logo from './logo.svg';
+import BooksList from './containers/BooksList';
+import BooksForm from './containers/BooksForm';
+// import logo from './logo.svg';
```

Comment:
- [ ] <b>REQUIRED:</b> This commented code here is useless in this context. Kindly remove it to maintain the clean code.

Reply:
Done, thanks